translations/en/LC_MESSAGES/messages.po

```diff
@@ -1,168 +1,86 @@
-# English translations for PROJECT.
-# Copyright (C) 2022 ORGANIZATION
-# This file is distributed under the same license as the PROJECT project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
-#
-msgid ""
-msgstr ""
-"Project-Id-Version: PROJECT VERSION\n"
-"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2023-10-17 19:06+0200\n"
-"PO-Revision-Date: 2023-10-01 00:33+0000\n"
-"Last-Translator: Yılmaz Durmaz <yilmaz_durmaz@hotmail.com>\n"
-"Language: en\n"
-"Language-Team: en <LL@li.org>\n"
-"Plural-Forms: nplurals=2; plural=n != 1;\n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.11.0\n"
-
 msgid "Access Before Assign"
-msgstr ""
-"You tried to use the variable {name} on line {access_line_number}, but "
-"you set it on line {definition_line_number}. Set a variable before using "
-"it."
+msgstr "You tried to use the variable {name} on line {access_line_number}, but you set it on line {definition_line_number}. Set a variable before using it."
```

Comment:
I am not sure why all these weird changes happen, I think @hasan-sh had a similar issue, do you remember that?